### ISSUE TITLE
Support for gedit

### ISSUE THREAD

**Issue by k4j8** [oskarsh/Yin-Yang#33]
I have created bash scripts to change programs between light and dark mode. I'm listing lines from these scripts with the intention of adding support for these programs once I understand how Yin-Yang is structured. If someone else wants to add it first, be my guest. The commands below have only been tested on Fedora Linux.

To change gedit to light mode:
`gsettings set org.gnome.gedit.preferences.editor 'scheme' 'solarized-light'`

To change gedit to dark mode:
`gsettings set org.gnome.gedit.preferences.editor 'scheme' 'solarized-dark`

**Comment by daehruoydeef** [comment]
I am thinking on how to implement this, there are several problems I am now facing since the App will react quite differently on GTK based Desktops in contrast to KDE. I would need to load different GUI´s for differnt operating systems. For example this is a Gnome only solution.

**Comment by KyleWJohnston** [comment]
Why not run changes for all known desktop environments? Running the Gnome script on KDE, or if Gedit is not even installed, won't mess anything up.

**Comment by daehruoydeef** [comment]
Yes you are right it will not mess it up but if I keep adding features for both desktops it will bloat up the ui, also the behavior is a bit different between those 2 platforms. For Example take the GTK Option, it does different things on Gnome in comparison to KDE. 

What I can offer for now if you want Gedit and all those other goodies included is to add a line to the config file. This way there will be no GUI option but users could manually insert the theme into the config and have Yin-Yang toggle it. I build it to be easily extended by plugins.
 You will need to edit the source code but I can help you out with that.

Basically check out this Pull Request that got merged a while ago that add Gnome Shell Theme to the config.
[Pull Request to add Gnome Shell Theme](https://github.com/daehruoydeef/Yin-Yang/pull/43)
check out the commit, it´s really easy to understand.
[https://github.com/daehruoydeef/Yin-Yang/pull/43/commits/19a5c9ef6312d85553bcde052738f3106d1409bb](https://github.com/daehruoydeef/Yin-Yang/pull/43/commits/19a5c9ef6312d85553bcde052738f3106d1409bb)
